platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -19,6 +19,11 @@
          name="Web Context Policy Manager"
          id="org.codice.ddf.security.policy.context.impl.PolicyManager">
 
+        <AD description="Depth to which paths will be traversed. Any value greater than 500 will be set to 500."
```

Comment:
I am inclined to agree with @coyotesqrl . Not that it matters much, but `100` is a more conventional arbitrary limit in my experience as well.